examples/advanced/tidb-cluster.yaml

```diff
@@ -244,8 +244,10 @@ spec:
   # Whether enable PVC reclaim for orphan PVC left by statefulset scale-in
   enablePVReclaim: false
 
-  # Persistent volume reclaim policy applied to the PVs that consumed by TiDB cluster
-  pvReclaimPolicy: Recycle
+  ## Persistent volume reclaim policy applied to the PVs that consumed by TiDB cluster, default is Retain
+  ## should be set to Retain when use local volume provisioner.
+  ## Ref: https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/
+  # pvReclaimPolicy: Retain
```

Comment:
```suggestion
  pvReclaimPolicy: Retain
```

Reply:
I think we can explicitly set it here since it's the recommended config for production. WDYT @weekface @cofyc

Reply:
I'm fine with either way.

Reply:
agree